react-native-ssi/example/src/App.tsx

```diff
@@ -28,20 +30,22 @@ export function App() {
               style={styles.button}
               onPress={() => {
                 generateDidKey('RSA').then((result) => {
-                  addLogLine(result.didKey);
                   setDidKey(result.didKey);
+                  addLogLine(didKey);
+
                   setPrivateJwk(result.privateJwk);
+                  addLogLine(privateJwk);
```

Comment:
linter demands that we use every state (this is fair and will catch bugs in the real world)

Reply:
This will just print `"undefined"` every time. Is there a way we can make it print something useful?